packages/prop-house-webapp/src/components/pages/FAQ/index.tsx

```diff
@@ -31,22 +35,62 @@ const content: ContentItem[] = [
 ];
 
 const FAQ = () => {
+  const [input, setInput] = useState('');
+  const [filteredFAQs, setfFilteredFAQs] = useState(content);
+
   const { t } = useTranslation();
 
+  const handleFAQInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInput(e.target.value);
+  };
+
+  useEffect(() => {
+    if (input.length === 0) setfFilteredFAQs(content);
+
+    setfFilteredFAQs(
+      content.filter(c => {
+        const query = input.toLowerCase();
+
+        return t(c.title.toLowerCase()).indexOf(query) >= 0;
+      }),
+    );
+  }, [input, t]);
+
   return (
     <>
-      <Container>
-        <h1 className={classes.title}>{t('frequentlyAsked')}</h1>
-        <Accordion flush className={classes.accordion}>
-          {content.map((item, i) => (
-            <div key={item.title}>
-              <Accordion.Item eventKey={`${i}`} className={classes.accordionItem}>
-                <Accordion.Header>{t(content[i].title)}</Accordion.Header>
-                <Accordion.Body>{ReactHtmlParser(t(`${content[i].content}`))}</Accordion.Body>
-              </Accordion.Item>
-            </div>
-          ))}
-        </Accordion>
+      <div className="faqGradientBg">
+        <Container>
+          <NavBar />
+
+          <div className={classes.searchWrapper}>
+            <h1 className={classes.title}>{t('frequentlyAsked')}</h1>
+
+            <SearchBar
+              input={input}
+              handleSeachInputChange={handleFAQInputChange}
+              placeholder="Search for questions"
+            />
+          </div>
+        </Container>
+      </div>
+
+      <Container className={classes.faqBackground}>
+        {filteredFAQs.length === 0 ? (
+          <RoundMessage message="No FAQs found" />
```

Comment:
Rename component to be generalized such that it can be used here without causing confusion.